modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/constants/WorkflowDefinitionConstants.java

```diff
@@ -12,4 +12,10 @@ public class WorkflowDefinitionConstants {
 
 	public static final String SCOPE_ALL = "all";
 
+	public static final String MESSAGE_BOARDS_USER_STATS_MODERATION_ERC =
+		"L_MESSAGE_BOARDS_USER_STATS_MODERATION";
+
+	public static final String SINGLE_APPROVER_ERC =
+		"L_SINGLE_APPROVER";
```

Comment:
I searched a little more and I found a pattern in the portal. I want to propose the following:

```
public static final String
EXTERNAL_REFERENCE_CODE_MESSAGE_BOARDS_USER_STATS_MODERATION =
	"L_MESSAGE_BOARDS_USER_STATS_MODERATION";

public static final String EXTERNAL_REFERENCE_CODE_SINGLE_APPROVER =
	"L_SINGLE_APPROVER";
```

Reply:
Please, remove the `MBModerationConstants` class and add it to the `WorkflowDefinitionConstants`:

```
public static final String NAME_CODE_MESSAGE_BOARDS_USER_STATS_MODERATION =
	"message-boards-user-stats-moderation";
```

Be careful to replace all the usages of `MBModerationConstants.WORKFLOW_DEFINITION_NAME` with the new one.